schema/schema.sql

```diff
@@ -0,0 +1,66 @@
+DROP TABLE IF EXISTS purchase;
+DROP TABLE IF EXISTS item_genre_assignment;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS item;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS format;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS subscriber;
+
+
+CREATE TABLE subscriber (
+    subscriber_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    name VARCHAR(255) UNIQUE NOT NULL
```

Comment:
People can have the same names so this cannot be unique

Reply:
Good spot, I agree! Fixing this now.